Tools/TestWebKitAPI/Tests/WebKitCocoa/WKWebExtensionWindow.mm

```diff
@@ -38,6 +38,12 @@
 #import <WebKit/_WKWebExtensionPrivate.h>
 #import <WebKit/_WKWebExtensionWindow.h>
 
+@interface TestWindow : NSObject <_WKWebExtensionWindow>
```

Comment:
i think this is a merge conflict error. will remove

Reply:
Yeah I recently remove this class.